sky/unit/test/engine/append_child_test.dart

```diff
@@ -10,7 +10,7 @@ void main() {
   test("should throw with invalid arguments", () {
     var parent = document.createElement("div");
     expect(() {
-      parent.appendChild();
+      Function.apply(parent.appendChild, []);
```

Comment:
Clever.

You can probably remove the ".analyzer_options" file that told the analyzer to skip this file.